scripts/convert_stl_to_obj.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import os
+import trimesh
+import argparse
+
+
+def convert_stl_to_obj(directory):
+    """
+    Finds all STL files in the given directory and subdirectories, and converts each to an OBJ file.
+
+    Args:
+        directory (str): The path to the directory to search.
+    """
+    for root, _, files in os.walk(directory):
+        for file in files:
+            stl_path = os.path.join(root, file)
+            file_root, file_ext = os.path.splitext(stl_path)
+
+            if file_ext.lower() in [".stl", ".STL"]:  # Check if the file is an STL file
```

Comment:
not necessary to use this list in this case since you're already converting the extension to lowercase